check_state.pm

```diff
@@ -1068,8 +1052,8 @@ sub pline04t {
 
 sub aacccdpt {
   my $val = $_[0];
-  my $afile = "./.aacccdptalert";
-  my $tfile = "./.aacccdptwait";
+  my $afile = ".aacccdptalert";
+  my $tfile = ".aacccdptwait";
```

Comment:
if we are no longer sending alerts for high temperature ccd then do we need this function?

Reply:
This is to be decided. Similarly, we also no longer send alerts for tank pressure.